### DISCUSSION TITLE
der-eine / ioBroker-LichtUeberwachung

### DISCUSSION THREAD

**Discussion by actionjackson09** [community/community#31080] [Other Feature Feedback, Questions, & Ideas]
Hallo zusammen, es gibt hier auf Github die LichtUeberwachung. Meine Frage dazu ist, welche Zeilen man deaktivieren könnte, um die Grafische Liste auszuschalten. Sollte ein Licht an sein, würde es mir völlig ausreichen, wenn dies als Text angezeigt wird. Dies macht das Script ja bereits allerdings mit zusätzlicher Grafischer Darstellung.
BG

**Comment by jfullstackdev** [comment]
as much as we want to help but if you can translate it to English using an online translator, that will be better

**Reply by actionjackson09** [reply]
Hi everyone, there is LightWatch here on Github. My question about it is, which lines could be disabled to turn off the graphical list. If a light should be on, it would be completely sufficient for me if this is displayed as text. The script already does this but with additional graphical display.
best regards

**Reply by airtower-luna** [reply]
Then you should create an issue or discussion in their repository. This is the general community, if any of the LightWatch developers sees your question here it's pure luck. :wink:

**Reply by actionjackson09** [reply]
okay i thought that someone who knows this programming language can just skim over it and know what to exclude.
Best regards
